src/VisualStudio/Core/Impl/CodeModel/InternalElements/CodeParameter.cs

```diff
@@ -62,6 +62,17 @@ protected override EnvDTE.CodeElements GetCollection()
             return GetCollection<CodeParameter>(Parent);
         }
 
+        protected override string GetName()
+        {
+            return this._name;
+        }
+
+        protected override string GetFullName()
+        {
+            var node = LookupNode();
```

Comment:
Yes, it'll throw if the parameter is removed and a consumer is still holding onto the EnvDTE.CodeParameter. This is a problem throughout Code Model.